src/main/kotlin/org/jitsi/nlj/dtls/DtlsUtils.kt

```diff
@@ -56,7 +56,7 @@ class DtlsUtils {
             val digAlgId = DefaultDigestAlgorithmIdentifierFinder().find(hashFunction.toUpperCase())
             val digest = BcDefaultDigestProvider.INSTANCE.get(digAlgId)
             val input: ByteArray = certificate.getEncoded(ASN1Encoding.DER)
-            val output: ByteArray = ByteArray(digest.digestSize)
+            val output = ByteArray(digest.digestSize)
```

Comment:
was there a reason to change this?  not exactly consistent here with the typing i guess (and maybe we should be) but just wondering

Reply:
The IDE suggested it, and shorter seemed better. Want me to bring it back?

Reply:
no, i guess it's clear actually since we very clearly call the ctor there (unlike `input` where it's assigned to a return value)